string classifier_name
uint32 classifier_id
uint32[] votes
uint32[] classes
